deid/bene_id_mapping.sql

```diff
@@ -127,12 +102,10 @@ create table pmap_parts (
 alter table pmap_parts parallel (degree 12);
 
 insert /*+ APPEND */ into pmap_parts
-select /*+ PARALLEL(bene_id_mapping,12) PARALLEL(msis_person,12) PARALLEL(msis_id_mapping,12) */
-  bmap.bene_id_deid, mmap.msis_id_deid, mper.state_cd, 
+select /*+ PARALLEL(bene_id_mapping,12) */
+  bmap.bene_id_deid, null msis_id_deid, null state_cd, 

```

Comment:
I don't understand this change. What is it for?

Reply:
I ll get rid of that sql statement.

Original SQL statement was being used to generate `patient_num`  if  `bene_id_deid`  does not exist for Medicaid data,
As we don't have Medicaid data, I can get rid off it, but it is optional to get rid of the statement as It does not have side effects besides increasing run time.

Reply:
fixed it at 01fbe46